### ISSUE TITLE
Invitation to move to official `eslint-community` org

### ISSUE THREAD

**Issue by MichaelDeBoey** [mysticatea/eslint-plugin-eslint-comments#72]
We would love to have this repo added to the official [`@eslint-community` organization on GitHub](https://github.com/eslint-community).

As you can read in [the '`@eslint-community` GitHub organization' RFC](https://github.com/eslint/rfcs/tree/main/designs/2022-community-eslint-org), the goal of this new org is to have a place where community members can help ensure widely depended upon ESLint related packages stay up to date with newer ESLint releases and doesn't hold the wider community back without depending on one person's GitHub/`npm` account.

Since this plugin is really popular (1M+ download/week) & since it's currently unmaintained (the latest commit is 2y+ old & the latest interaction is 2y+ old as well), we'd love you —@mysticatea— to transfer this repo to a better home, so you're welcome to transfer this repository to the new org.

**Comment by nmschulte-aviture** [comment]
@mysticatea do it!  This valuable project will see ESLint v8 support there I imagine: https://github.com/mysticatea/eslint-plugin-eslint-comments/issues/62

**Comment by nmschulte-aviture** [comment]
@mysticatea do it!  This valuable project will see ESLint v8 support there I imagine: https://github.com/mysticatea/eslint-plugin-eslint-comments/issues/62

@MichaelDeBoey, the author notes the project is MIT licensed, and supports your incorporation into the community and other projects: https://github.com/mysticatea/eslint-plugin-eslint-comments/issues/65#issuecomment-964891049

**Comment by user72356** [comment]
I think dude is a goner TBH

**Comment by moroine** [comment]
This repo doesn't seems to be maintained anymore 😢 I'd like to see this repo support ESlint8, and looks like the PR was opened last year https://github.com/mysticatea/eslint-plugin-eslint-comments/pull/63

**Comment by user72356** [comment]
Is there no other eslint rule out there that support detection of es-lint-disable comments?

**Comment by MichaelDeBoey** [comment]
@moroine We started forking some of @mysticatea's repos already in the @eslint-community org and we plan to fork this as well if we don't receive any response.
Of course we would love to move the original repo over, but we're starting with forking the repos so the wider community isn't held back any longer

**Comment by moroine** [comment]
@MichaelDeBoey could you clone the repo, so I can open PR to help?

**Comment by MichaelDeBoey** [comment]
@moroine We're forking repos one by one & taking our time to update everything properly.
I'll ping everybody here once we've forked `eslint-plugin-eslint-comments`

**Comment by christianvuerings** [comment]
@MichaelDeBoey thank you for forking the project to https://github.com/eslint-community/eslint-plugin-eslint-comments !

Do you have a guess as to when we might see the first publish of `@eslint-community/eslint-plugin-eslint-comments`?

**Comment by nmay231** [comment]
@christianvuerings https://www.npmjs.com/package/@eslint-community/eslint-plugin-eslint-comments

**Comment by silverwind** [comment]
I hope some of the PRs there could be merged, currently it seems like a mere version bump.

**Comment by MichaelDeBoey** [comment]
@silverwind There was never any PR of you on the @eslint-community repo
https://github.com/eslint-community/eslint-plugin-eslint-comments/pulls/silverwind

**Comment by silverwind** [comment]
Yes I'm guilty of not contributing, but I was more talking about https://github.com/eslint-community/eslint-plugin-eslint-comments/pull/20 😉.
